velox/docs/functions/spark/array.rst

```diff
@@ -151,6 +151,14 @@ Array Functions
         SELECT array_sort(array(NULL, 1, NULL)); -- [1, NULL, NULL]
         SELECT array_sort(array(NULL, 2, 1)); -- [1, 2, NULL]
 
+.. spark:function:: array_union(array(E), array(E1)) -> array(E2)
```

Comment:
Does E means the data type? The data type should be same. See `array_sort` and `arrays_zip` as example.